src/composables/useTooltips.ts

```diff
@@ -0,0 +1,44 @@
+import { storeToRefs } from "pinia";
+import { useAppStore } from "../stores/appStore";
+
+export const useTooltips = () => {
+    const { selectedIndicators, selectedIndicator, appConfig } = storeToRefs(useAppStore());
+
+    const sortedIndicators = computed(() => {
+        const sortedKeys = Object.keys(appConfig.value.indicators).sort((indicatorName) =>
+            indicatorName.toLowerCase() === selectedIndicator.value.toLowerCase() ? -1 : 1
+        );
```

Comment:
This is sorting on the indicator id, but it would be better to sort on human readable name since that's what we're displaying.

Reply:
Oh yes, I need to make sure that the ids match, not compare a name with an id.

Reply:
In fact, this code was correctly comparing ids/keys with ids/keys (no names involved); so I'm just renaming the param from 'indicatorName' to 'indicatorId'.

Reply:
(NB this isn't an alphabetical sort, it's sorting the selected indicator to the top of the list.)